From 9c6fb372191e19597fdfd2ac417b00e9396e7c54 Mon Sep 17 00:00:00 2001
From: intrigeri <intrigeri@boum.org>
Bug-Debian: https://bugs.debian.org/831384
Bug-Tails: https://labs.riseup.net/code/issues/10298#note-19
Date: Fri, 15 Jul 2016 11:54:39 +0000
Subject: [PATCH] Don't force using the vboxvideo X.Org driver.

According to our tests, this forced setting is:

 * harmful: it breaks X startup when the vboxvideo *kernel* driver
   is loaded;

 * useless: X.Org now autodetects the vboxvideo X.Org driver and uses it
   when running in VirtualBox and the vboxvideo kernel is not present.

Note that I'm not touching the InputDevice section here, since I don't
know if it is still useful, and have not heard of any problem with it.

Closes: #831384
---
 share/xserver-xorg/vboxvideo.conf | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/share/xserver-xorg/vboxvideo.conf b/share/xserver-xorg/vboxvideo.conf
index 4d7a966..dc4f4d2 100644
--- a/usr/share/live/config/xserver-xorg/vboxvideo.conf
+++ b/usr/share/live/config/xserver-xorg/vboxvideo.conf
@@ -1,8 +1,3 @@
-Section "Device"
-	Identifier	"Default screen"
-	Driver		"vboxvideo"
-EndSection
-
 Section "InputDevice"
 	Identifier	"Default mouse"
 	Driver		"vboxmouse"
-- 
2.8.1

